.github/sync-files.yaml

```
@@ -1,30 +1,36 @@
-- repository: autowarefoundation/autoware
+- repository: autowarefoundation/sync-file-templates
+  source-dir: sources
   files:
-    - source: CODE_OF_CONDUCT.md
-    - source: CONTRIBUTING.md
-    - source: DISCLAIMER.md
-    - source: LICENSE
     - source: .github/ISSUE_TEMPLATE/bug.yaml
     - source: .github/ISSUE_TEMPLATE/config.yml
     - source: .github/ISSUE_TEMPLATE/task.yaml
-    - source: .github/PULL_REQUEST_TEMPLATE.md
-    - source: .github/PULL_REQUEST_TEMPLATE/small-change.md
-    - source: .github/PULL_REQUEST_TEMPLATE/standard-change.md
     - source: .github/dependabot.yaml
+    - source: .github/pull_request_template_complex.md
+      dest: .github/pull_request_template.md
```

```suggestion
    - source: .github/pull_request_template.md
```

I think we can use the simple one here

---

fixed in https://github.com/autowarefoundation/awsim_labs_sensor_kit_launch/pull/13/commits/f2d2e6029f103cdbb931b625784575ee74c06b2a